dashboard/helpers.py

```diff
@@ -13,3 +14,17 @@ def get_natural_size(value):
     str_value = humanize.naturalsize(value, format='%.2f', binary=True)
     # KiB -> KB, MiB - > MB, ..
     return str_value.replace('i', '')
+
+
+def format_duration(duration: timedelta) -> str:
+    result = ''
+    total_seconds = duration.total_seconds()
+    if total_seconds < 60:
+        result = '< 1 minute'
```

Comment:
just short circuit this, aka 
```python
 if total_seconds < 60:
        return '< 1 minute'
```

no need for else then

Reply:
changed